core/domain/suggestion_services.py

```diff
@@ -318,10 +395,26 @@ def reject_question_suggestions_with_skill_target_id(skill_id):
             ('target_id', skill_id)
         ]
     )
-    for suggestion in suggestions:
-        reject_suggestion(
-            suggestion, feconf.SUGGESTION_BOT_USER_ID,
-            suggestion_models.DELETED_SKILL_REJECT_MESSAGE)
+    reject_suggestions(
+        suggestions, feconf.SUGGESTION_BOT_USER_ID,
+        suggestion_models.DELETED_SKILL_REJECT_MESSAGE)
+
+
+def reject_translation_suggestions_with_exp_target_ids(exp_ids):
```

Comment:
Maybe autoreject_* or anything similar to highlight that this method will reject suggestions from a bot account*